pkg/api/resource/quantity.go

```diff
@@ -287,7 +294,7 @@ func ParseQuantity(str string) (Quantity, error) {
 	switch format {
 	case DecimalExponent, DecimalSI:
 		scale = exponent
-		precision = 15 - int32(len(num)+len(denom))
+		precision = 18 - int32(len(num)+len(denom))
```

Comment:
Will iterate tonight, squash, and reorganize the commits.  I think I have
tests for all the code paths covered (specifically the Add / Sub stress
test), Cmp, and the existing String tests.

On Mon, May 16, 2016 at 8:17 PM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> Probably a different one - "maxDecimalPlacesInBase10Int64".
>
> On Mon, May 16, 2016 at 7:57 PM, Daniel Smith <notifications@github.com>
> wrote:
>
>> In pkg/api/resource/quantity.go
>> <https://github.com/kubernetes/kubernetes/pull/25243#discussion_r63446356>
>> :
>>
>> > @@ -287,7 +294,7 @@ func ParseQuantity(str string) (Quantity, error) {
>> >  	switch format {
>> >  	case DecimalExponent, DecimalSI:
>> >  		scale = exponent
>> > -		precision = 15 - int32(len(num)+len(denom))
>> > +		precision = 18 - int32(len(num)+len(denom))
>>
>> should this 18 be the constant you made?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kubernetes/kubernetes/pull/25243/files/f8f37c149266b823a4a54d1b0efc21bf8376080b..2a61025932c283f19dcc4e003cc623c57cea71e0#r63446356>
>>
>
>